java/src/main/resources/template.properties

```diff
@@ -182,6 +182,12 @@ pubsubtogcs.batch.size=1000
 ## PubSub to GCS supported formats are: AVRO, JSON
 pubsubtogcs.gcs.output.data.format=
 
+#Hbase to GCS
```

Comment:
Please supply default values as value to there properties.
And rest of possibiliets in a comment.